### ISSUE TITLE
Curtailment in SA

### ISSUE THREAD

**Issue by BevDan** [opennem/opennem#391]
Not sure it is feasible, but has there been any thought of graphing / reporting the amount of curtailment being exercised each day, it may be a very SA problem, but by looking at all the various renewable energy generators in South Australia you can see which ones have reduced or switched off their output during the day as we can't export any more energy via the interconnector

**Comment by nc9** [comment]
@BevDan yes we have - we just happen to be talking about it yesterday. There's an issue on the open electricity repository:

https://github.com/opennem/openelectricity/issues/41

I'll keep this open here as it'll require some work, and I'll figure out exactly when we may be able to get to it. Thanks for logging an issue!

**Comment by nc9** [comment]
see #433 - this is queued up for the next release

**Comment by BevDan** [comment]
@nc9 Absolutely fantastic, well done on making this information available on the webpage
